src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -125,9 +125,39 @@ internal static bool IsPrimitiveType(this CorElementType et)
 
         [Intrinsic]
         internal static bool IsKnownConstant(char t) => false;
-
-        [Intrinsic]
-        internal static bool IsKnownConstant(int t) => false;
 #pragma warning restore IDE0060
+
+#if NATIVEAOT
+        [System.Runtime.RuntimeExport("RhRuntimeHelpers_MemCopy")]
```

Comment:
Can we export `SpanHelpers.Memmove` as JIT helper directly without going through the wrapper?

(Same for the other helpers in this file.)

Reply:
One tiny problem that implicit nullchecks (in front of pinvoke) will potentially slightly regress perf for all uses (not sure the cost of those will be noticeable compared to pinvoke machinery)

Reply:
Right, PInvoke path has quite a bit of overhead. An extra instruction in that path won't be measurable.